demos/button.html

```diff
@@ -66,345 +80,116 @@
     <main class="mdc-toolbar-fixed-adjust">
 
       <section class="hero">
-        <button class="mdc-button button">Flat</button>
-        <button class="mdc-button mdc-button--raised button mdc-button--primary">Raised</button>
+        <button class="mdc-button button">
```

Comment:
It was because I modified the file from the original BREAKING CHANGE pr with string substitution, I don't feel this make much of a difference though?